spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/CaffeLoader.scala

```diff
@@ -16,7 +16,7 @@
 
 package com.intel.analytics.bigdl.utils
 
-import java.io.{FileInputStream, InputStreamReader}
+import java.io._
```

Comment:
import java.io.{FileInputStream, InputStreamReader} is replaced by import java.io._